Automattic/wc-calypso-bridge
# Rename "Blog" frontend navigation item to "News"

### Changes proposed in this Pull Request:

Closes #1375  .


As discussed in https://github.com/Automattic/wc-calypso-bridge/issues/1321#issuecomment-1838235450

we'll rename Blog to News. Blog is coming from headstart
https://github.com/Automattic/theme-tsubaki/blob/trunk/inc/headstart/en.json#L279




### How to test the changes in this Pull Request:

1. Checkout this branch
2. If a Blog page (with slug blog) does not exist, create it
3. Comment out the following lines in `includes/class-wc-calypso-bridge-setup.php`

```php

public function woocommerce_create_pages_callback() {

			// Set the operation as completed if the store is active for more than 60 minutes.
			if ( WCAdminHelper::is_wc_admin_active_for( 60 * MINUTE_IN_SECONDS ) ) {
				update_option( $this->option_prefix . $operation, 'completed', 'no' );
				$this->write_to_log( $operation, 'completed (60 minutes)' );

				return;
			}
```

4. Rsync calypso bridge to your WoA dev site
5. SSH into the site, `cd htdocs` and then `wp option delete wc_calypso_bridge_one_time_operation_woocommerce_create_pages` 
6. Visit `wp-admin/edit.php?post_type=page` and see that Blog has been renamed to News
7. Visit the frontend, and see that the menu item is News
8. Visit the page, and see that the title is changed to News
